front/src/styles/scss/applications/stdcm/_results.scss

```diff
@@ -159,6 +159,9 @@
             padding-bottom: 0.313rem;
             vertical-align: middle;
           }
+          th:last-child {
+            text-transform: none;
```

Comment:
I'd prefer not to have a special case for the only header which happens to have multiple words: in the future, if one of the other translations get updated with a string which contains multiple words, `text-transform: capitalize` will not do what we want.

So maybe we can drop `text-transform: capitalize` and always just capitalize everything the way we want in the translation files?

Reply:
We use these translations in another file and we need them to be in lowercase there :(

Reply:
Maybe we can use `text-transform: lowercase` for the other place where the translations are used?

Reply:
This will be done in a future PR refactor of the simulation sheet

Reply:
Alright!